src/clr-core/internal/services/focus-trap-tracker.service.ts

```diff
@@ -0,0 +1,25 @@
+/*
+* Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+* This software is released under MIT license.
+* The full license information can be found in LICENSE in the root directory of this project.
+*/
+
+/**
+ * FocusTrapTracker is a static class that keeps track of the active element with focus trap,
+ * in case there are multiple in a given page.
+ */
+export class FocusTrapTracker {
+  private static focusTrapElements: Array<any> = [];
```

Comment:
nit: I see some element variable/properties are using `any` or `HTMLElement`. Should we just stick to `HTMLElement`?

Reply:
I'll change it to type `Element` 👍